projects/1.18/assets/essentials/essentials/lang/zh_cn.json

```
@@ -1 +1,222 @@
-{}
\ No newline at end of file
+{
+	"block.essentials.slotted_chest": "分格箱子",
+	"block.essentials.sorting_hopper": "分类漏斗",
+	"block.essentials.speed_hopper": "高通量漏斗",
+	"block.essentials.fertile_soil_wheat": "小麦沃土",
+	"block.essentials.fertile_soil_carrot": "胡萝卜沃土",
+	"block.essentials.fertile_soil_potato": "马铃薯沃土",
+	"block.essentials.fertile_soil_beetroot": "甜菜根沃土",
+	"block.essentials.fertile_soil_netherwart": "下界疣沃土",
+	"block.essentials.fertile_soil_oak": "橡树树苗沃土",
+	"block.essentials.fertile_soil_birch": "白桦树苗沃土",
+	"block.essentials.fertile_soil_spruce": "云杉树苗沃土",
+	"block.essentials.fertile_soil_jungle": "丛林树苗沃土",
+	"block.essentials.fertile_soil_dark_oak": "深色橡树树苗沃土",
+	"block.essentials.fertile_soil_acacia": "金合欢树苗沃土",
+	"block.essentials.fertile_soil_berry": "甜浆果沃土",
+	"block.essentials.fertile_soil_red_mushroom": "红色蘑菇沃土",
+	"block.essentials.fertile_soil_brown_mushroom": "棕色蘑菇沃土",
+	"block.essentials.hopper_filter": "漏斗过滤器",
+	"block.essentials.candle_lilypad": "蜡烛睡莲",
+	"block.essentials.brazier": "火盆",
+	"block.essentials.item_chute": "运输槽",
+	"block.essentials.item_shifter": "物品运输器",
+	"block.essentials.fluid_shifter": "流体运输器",
+	"block.essentials.multi_piston_sticky": "黏性倍能活塞",
+	"block.essentials.multi_piston": "倍能活塞",
+	"block.essentials.multi_piston_extend_sticky": "伸长的黏性倍能活塞",
+	"block.essentials.multi_piston_extend": "伸长的倍能活塞",
+	"block.essentials.basic_item_splitter": "基础物品分路器",
+	"block.essentials.item_splitter": "物品分路器",
+	"block.essentials.basic_fluid_splitter": "基础流体分路器",
+	"block.essentials.fluid_splitter": "流体分路器",
+	"block.essentials.wither_cannon": "凋灵加农炮",
+	"block.essentials.auto_crafter": "自动合成器",
+	"block.essentials.redstone_transmitter": "红石信号发射器",
+	"block.essentials.redstone_receiver": "红石信号接收器",
+	"block.essentials.analog_lamp": "模拟灯",
+
+	"block.essentials.wire_circuit": "电路接线板",
+	"block.essentials.wire_junction_circuit": "电路交叉点",
+	"block.essentials.interface_circuit": "电路接口板",
+	"block.essentials.and_circuit": "与电路",
+	"block.essentials.cons_circuit": "常量电路",
+	"block.essentials.not_circuit": "非电路",
+	"block.essentials.or_circuit": "或电路",
+	"block.essentials.xor_circuit": "异或电路",
+	"block.essentials.max_circuit": "最大值电路",
+	"block.essentials.min_circuit": "最小值电路",
+	"block.essentials.sum_circuit": "加法电路",
+	"block.essentials.dif_circuit": "减法电路",
+	"block.essentials.prod_circuit": "乘法电路",
+	"block.essentials.quot_circuit": "除法电路",
+	"block.essentials.pow_circuit": "求幂电路",
+	"block.essentials.inv_circuit": "倒数电路",
+	"block.essentials.sin_circuit": "Sin电路",
```

```suggestion
	"block.essentials.sin_circuit": "正弦电路",
```
理由是：psi、betterfps都将sine直接译为正弦，后文也使用了正弦一词。其余三角函数同理。
（这行我记得发出来了，不知道为什么并没有）